superset/cli.py

```diff
@@ -212,23 +211,23 @@ def import_dashboards(path, recursive, username):
     """Import dashboards from JSON"""
     from superset.utils import dashboard_import_export
 
-    p = Path(path)
+    pth = Path(path)
     files = []
-    if p.is_file():
-        files.append(p)
-    elif p.exists() and not recursive:
-        files.extend(p.glob("*.json"))
-    elif p.exists() and recursive:
-        files.extend(p.rglob("*.json"))
+    if pth.is_file():
+        files.append(pth)
+    elif pth.exists() and not recursive:
+        files.extend(pth.glob("*.json"))
+    elif pth.exists() and recursive:
+        files.extend(pth.rglob("*.json"))
     if username is not None:
         g.user = security_manager.find_user(username=username)
-    for f in files:
-        logging.info("Importing dashboard from file %s", f)
+    for _file in files:
```

Comment:
nit: `file_`?

Reply:
`f` is pretty standard as a generic file variable name, kind of like `l = []`. I'd vote to add to the list of allowed short variable names